game-core/build.gradle

```diff
@@ -26,12 +18,6 @@ def remoteFile(url) {
     files(file)
 }
 
-jar {
-    manifest {
-        attributes 'Main-Class': mainClassName, 'TripleA-Version': version
```

Comment:
We might want to store the version string inside the manifest, I believe it's currently the only thing the game properties file is being used for.
Also this makes the game version kinda immutable in the sense that you can simply change a value in the config file.
Just a thought.
(If course we'd need to store the version string somewhere else for general development)

Reply:
Yeah, I couldn't find any use of this manifest attribute, so I removed it.  (Same thing in the `game-headless` project.)  Do you have any history on it?  It should either be in every artifact or none of them.

> I believe it's currently the only thing the game properties file is being used for.

_game_engine.properties_ is read at runtime and used to initialize `ClientContext#engineVersion()`, which is used in a lot of places.  Were you talking about a different "game properties file"?

Reply:
No, this is exactly the file.
I think I didn't express myself good enough:
What I meant to say is that if we put the version string in the manifest/keep it there, we could actually use it and drop the game_engine properties file entirely, because AFAIK this file isn't being used for anything else anymore.
So we would change the code in client context to read the version string from the manifest for jars, and from somewhere dev-config-exclusive like gradle properties or similar otherwise.

Reply:
Understood and agreed.  That will also help us decouple the `game-headed` and `game-headless` projects from depending on running from the `game-core` project directory so they can find _game_engine.properties_.

There's probably a way to place the manifest (or a manifest fragment) under _src/main/resources_ so that we can always find it from the classpath, regardless of whether we're running from a jar or the file system (i.e. a dev environment).